codice/ddf
# DDF-4481 Add the IS EMPTY operator in search queries

#### What does this PR do?
Adding search capabilities for the new IS EMPTY operator which returns all metacards when the attribute itself does not exist, or if the attribute value is null.

#### Who is reviewing it? 

@derekwilhelm 
@bellcc 
@GabrielFabian 
@jlcsmith 
@andrewkfiedler 

#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@jlcsmith

#### How should this be tested?

Ingest data and then perform searches and ensure that the correct results are being returned.
Make sure that the ingested data has attributes that is null, or doesn't exist, or has the empty string or has legitimate values so that you can tell if the operator behaves as it should

#### Any background context you want to provide?
#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #____

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
